joomla/joomla-cms
# Remove unneeded metadata.xml files

Currently, we have metadata.xml files in each view. However since Joomla 3.5.0 those are no longer required (PR https://github.com/joomla/joomla-cms/pull/7654).
### Summary of Changes

This PR removes the metadata.xml files since they have no function at all in core.
This will help extension developers to see best practices.
### Testing Instructions
- Test creating menu items for the affected views. Nothing strange should happen.
- Check the menu item type text in the menu item manager, see picture. Nothing should change there.
  ![metadata](https://cloud.githubusercontent.com/assets/1018684/18362149/f2ed8492-7604-11e6-9f69-0e1122db06be.PNG)
### Documentation Changes Required

None to my knowledge.
